pydantic/validators.py

```diff
@@ -80,17 +80,20 @@ def bytes_validator(v: Any) -> bytes:
         raise errors.BytesError()
 
 
-BOOL_STRINGS = {'1', 'TRUE', 'ON', 'YES'}
+BOOL_TRUE = {True, 1, 'y', 'Y', 'yes', 'Yes', 'YES', 'true', 'True', 'TRUE', 'on', 'On', 'ON', '1', 't', 'T'}
```

Comment:
wouldn't it be simpler if we just did `v.lower() in BOOL_TRUE` and removed the case duplicates?

I get that this would mean `'trUE' -> True` but does anyone care?

Reply:
Yup I agree

Reply:
I'll change it